00-RELEASENOTES

```diff
@@ -1,16 +1,284 @@
-Hello! This file is just a placeholder, since this is the "unstable" branch
-of Valkey, the place where all the development happens.
+Valkey 8.0 release notes
+========================
+--------------------------------------------------------------------------------
+Upgrade urgency levels:
+LOW:      No need to upgrade unless there are new features you want to use.
+MODERATE: Program an upgrade of the server, but it's not urgent.
+HIGH:     There is a critical bug that may affect a subset of users. Upgrade!
+CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
+SECURITY: There are security fixes in the release.
+--------------------------------------------------------------------------------
 
-There is no release notes for this branch, it gets forked into another branch
-every time there is a partial feature freeze in order to eventually create
-a new stable release.
+================================================================================
+Valkey 8.0.0 RC1  -  Released Thu 1 Aug 2024
+================================================================================
+Upgrade urgency LOW: This is the first release candidate of Valkey 8.0, with
+performance, reliability, and observability improvements. It includes asynchronous
+I/O threading, better cluster scaling reliability, dual primary-replica channel
+for faster full synchronization, per-slot metrics for resource management, and
+experimental RDMA support for increased throughput and reduced latency. This
+release is fully compatible with OSS Redis 7.2.4.
```

Comment:
```suggestion
release is fully compatible with Redis OSS 7.2.4.
```
Probably overly paranoid, but this is how Redis describes it so usually go along with their naming.